tbx/core/models.py

```diff
@@ -258,3 +258,24 @@ class MainMenu(BaseSiteSetting):
     panels = [
         FieldPanel("menu"),
     ]
+
+
+class ServicePage(SocialFields, Page):
+    template = "patterns/pages/service/service_page.html"
+
+    intro = RichTextField(blank=True)
+    body = StreamField(ServiceStoryBlock(), use_json_field=True)
```

Comment:
I'm not certain, but I have a feeling that on the current version of wagtail we no longer need to add `use_json_field` - I think the default has now changed.

Reply:
From the [5.0 release notes](https://docs.wagtail.org/en/stable/releases/5.0.html#removal-of-deprecated-features), it seems we still need to:

> StreamField definitions must include `use_json_field=True` (with the exception of migrations created prior to Wagtail 5.0).